Sources/NIOHTTPClient/HTTPHandler.swift

```diff
@@ -94,6 +94,111 @@ public extension HTTPClient {
         public var headers: HTTPHeaders
         public var body: ByteBuffer?
     }
+
+    struct HTTPCookie {
```

Comment:
I think having in in a separate file would be better, we can have `extension HTTPClient` there, that will prevent `HTTPHandler.swift` from becoming overloaded.

Reply:
Cool, works for me. Think it should be called something like `HTTPClient+HTTPCookie.swift`?

Reply:
Yep, that will work!

Reply:
Lemme know what you think! 🙇‍♂️